### PR TITLE
add an option for 斗战胜佛.md and create the-right-way

### PR DESCRIPTION
**Pull request by JoyceChiang**

### CODE REVIEW

**Comment by JoyceChiang** [comment]
Please review this, @casey-collab

File: Chinese/the-right-way/to_be_a_coder.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+悟空在天庭当斗战胜佛当了很多年，这个体制内的差事一直稳定有油水，然而近些年的反贪腐运
+动轰轰烈烈地开展着，日子渐渐变得不好过起来。
+
+悟空听人说二十一世纪是生物的时代，但又看到互联网的风口中八戒都能飞起来，一时拿不定主
+意是转行进入
+[生物行业]()，
```
Comment (casey-collab) [inline Chinese/the-right-way/to_be_a_coder.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

File: Chinese/the-right-way/to_be_a_coder.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+[还是成为程序猿]()。
```
Comment (casey-collab) [inline Chinese/the-right-way/to_be_a_coder.md]: This link is also invalid.

Comment (casey-collab) [inline Chinese/the-right-way/to_be_a_coder.md]: There is a link on this line and also on the previous line.  Please make sure to insert a blank line between links.  That will help keep options visually separated.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by JoyceChiang** [comment]
Please have another look, @casey-collab

**Comment by casey-collab** [comment]
Looks great! :lollipop:

**Pull request merged**